src/WormholeL2GovernorMetadata.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
```

Comment:
We should also make an issue to bump this at some point

Reply:
created #46